com.valvesoftware.Steam.CompatibilityTool.Boxtron.yml

```diff
@@ -0,0 +1,91 @@
+id: com.valvesoftware.Steam.CompatibilityTool.Boxtron
+runtime: com.valvesoftware.Steam
+sdk: org.freedesktop.Sdk//19.08
+build-extension: true
+appstream-compose: false
+build-options:
+  prefix: /app/share/steam/compatibilitytools.d/Boxtron
+  prepend-path: /app/share/steam/compatibilitytools.d/Boxtron/bin
+  prepend-pkg-config-path: /app/share/steam/compatibilitytools.d/Boxtron/lib/pkgconfig
+  env:
+    ACLOCAL_PATH: /app/share/steam/compatibilitytools.d/Boxtron/share/aclocal
+    CPATH: :/app/share/steam/compatibilitytools.d/Boxtron/include
+  strip: true
+cleanup:
+  - "*.a"
+  - "*.la"
+  - /include
+  - /lib/pkgconfig
+  - /share/aclocal
+  - /share/man
+modules:
+
+  - name: dosbox
+    sources:
+      - type: archive
+        url: "https://downloads.sourceforge.net/project/dosbox/dosbox/0.74-3/dosbox-0.74-3.tar.gz"
+        sha256: c0d13dd7ed2ed363b68de615475781e891cd582e8162b5c3669137502222260a
+    modules:
+
+      - shared-modules/SDL/SDL-1.2.15.json
+
+      - shared-modules/SDL/SDL_net-1.2.8.json
+
+      - name: SDL_sound
+        sources:
+          - type: archive
+            url: "https://icculus.org/SDL_sound/downloads/SDL_sound-1.0.3.tar.gz"
+            sha256: 3999fd0bbb485289a52be14b2f68b571cb84e380cc43387eadf778f64c79e6df
+
+      - shared-modules/glu/glu-9.json
+
+  - name: libXinerama
+    sources:
+      - type: archive
+        url: "https://www.x.org/releases/individual/lib/libXinerama-1.1.4.tar.gz"
+        sha256: 64de45e18cc76b8e703cb09b3c9d28bd16e3d05d5cd99f2d630de2d62c3acc18
+
+  - name: fluidsynth
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DLIB_SUFFIX=
+    sources:
+      - type: archive
+        url: "https://github.com/FluidSynth/fluidsynth/archive/v2.0.6.tar.gz"
+        sha256: e97e63c1045e102465f1aa848f9d712c5528c58685b8d40062e4aaf6af7edb75
+
+  - name: fluid-soundfont
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 FluidR3_GS.sf2 -t ${FLATPAK_DEST}/share/sounds/sf2/
+      - ln -s FluidR3_GS.sf2 ${FLATPAK_DEST}/share/sounds/sf2/default.sf2
+    sources:
+      - type: archive
+        url: "http://deb.debian.org/debian/pool/main/f/fluid-soundfont/fluid-soundfont_3.1.orig.tar.gz"
+        md5: 189bbdf70221018cbda536984b105dfa
+
+  - name: inotify-tools
+    sources:
+      - type: archive
+        url: "https://downloads.sourceforge.net/inotify-tools/inotify-tools-3.13.tar.gz"
+        sha256: 464146acec2c118be63a6c2a090d9252fbe9689c093db85b695d0502fa229445
+
+  - name: Boxtron
+    #TODO enable linting, requires shellcheck and pylint
+    buildsystem: simple
```

Comment:
Would it be too much effort to build Boxtron ourselves against org.freedesktop.Sdk?

Reply:
It's written in python, and is kind of built from source here.

Reply:
Fair enough. Sounds good to me then.